azkaban_cli/azkaban_cli.py

```diff
@@ -147,6 +147,26 @@ def __create(ctx, project, description):
     except CreateError as e:
         logging.error(str(e))
 
+@login_required
+def __delete(ctx, project):
+    azkaban = ctx.obj[u'azkaban']
+    try:
+        # Since delete doesn't raise an exception, we try fetching the project flows.
+        # Project IDs get stored within the database even after deletion, so we then
+        # try to fetch the project schedule.
```

Comment:
@aguimaraesduarte If the project really doesn't have any schedule before the delete command, will it work as expected?

Reply:
@ggalves I looked over the code again and realized it wasn't working as expected in all scenarios. I rewrote the logic behind deleting a project so that it now unschedules the scheduled flows it may have before attempting to delete it. This seems to be working a lot better.
The only "issue" is that Azkaban saves the project IDs within its own internal database, leading to an edge case when the project has already been deleted or doesn't have any flows registered. There is no way to discern between the two alternatives, and the command will always say that it successfully deleted the project. I added a logging output to explain this to the user, should this happen.
Do you think this is an acceptable workaround?

Reply:
Well, not ideal but it is not your fault. The thing starts wrong when the delete command is a GET request :P 

The workaround looks good, for me it's fine.